scipy/interpolate/ndgriddata.py

```diff
@@ -132,6 +132,13 @@ def griddata(points, values, xi, method='linear', fill_value=np.nan,
         incommensurable units and differ by many orders of magnitude.
 
         .. versionadded:: 0.14.0
+        
+    Returns
+    -------
+    
+    ip: ndarray
```

Comment:
There is no need to create a name for the return value; this line can be simply `ndarray`.  See the subsection "5. Returns" in the file https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections